Willning/DLOplanTool
# DLO 13 sharing project

### **Related Issue/Keyword:**
DLO-13
### **Description:**
share a project to your friends on the mapping page and your friends will receive a notification
### **Testing:**

Add anything in particular to look out for

**Steps for manual testing:**

1. create a new project
2. click the "Share" button at the bottom right
3. input the emails(these emails should be existed in this app, currently can share to oneself) and click the "Share" button
4. click the "Notifications" button at the HeaderBar, you will see a notice list

### **Checklist:**

- [] Meets the projects coding conventions
